src/mem/cache/tags/fa_lru.cc

```diff
@@ -191,7 +192,8 @@ FALRU::findBlockBySetAndWay(int set, int way) const
 
 CacheBlk*
 FALRU::findVictim(Addr addr, const bool is_secure, const std::size_t size,
-                  std::vector<CacheBlk*>& evict_blks)
+                  std::vector<CacheBlk*>& evict_blks,
+                  const uint64_t partition_id)
```

Comment:
Something must be done with the partition id here, as the implementation seems pretty trivial (preferred). Either that or assert that PPs are not used when this tag type is used.

Reply:
Added `fatal` check in FALRU constructor if partitioning policies have been supplied; I think that enforcing PPs on FALRU caches is outside the scope of the PR.